test_manager/features/steps/tck_step_implementations.py

```diff
@@ -35,6 +36,7 @@
 PYTHON_TA_PATH = "/test_agent/python/testagent.py"
 JAVA_TA_PATH = "/test_agent/java/target/tck-test-agent-java-jar-with-dependencies.jar"
 RUST_TA_PATH = "/test_agent/rust/target/debug/rust_tck"
+CPP_TA_PATH = "/test_agent/cpp/target/bin/test_agent_cpp"
 DISPATCHER_PATH = "/dispatcher/dispatcher.py"
```

Comment:
`DISPATCHER_PATH` is defined, but I don't find the use. I'm wondering if this is typo

Reply:
I agree with you but this is outside scope of this PR. I have opened issue to address this

https://github.com/eclipse-uprotocol/up-tck/issues/55